CONTRIBUTING.md

```diff
@@ -0,0 +1,389 @@
+# How to contribute to Arcs
+
+Thank you for taking the time to contribute to the Arcs project. 
+This guide outlines how to become an Arcs developer.
+
+**Table of Contents**
+
+[Workflow](#workflow)
+* [Filing Bugs](#filing-bugs)
+* [Contributing a Patch](#contributing-a-patch)
+
+[Developer Installation](#developer-installation)
+* [Additional Bazel Installation](#additional-bazel-installation)
+* [Windows Installation Notes](#windows-installation-notes)
+* [Mac Installation Notes](#mac-installation-notes)
+* [IDE Setup](#ide-setup)
+* [Git Setup](#git-setup)
+
+[Starting Arcs](#starting-arcs)
+
+[Testing](#testing)
+* [WebAssembly tests](#webassembly-tests)
+* [Debugging tests](#debugging-tests)
+  * [Debugging unit tests in Chrome](#debugging-unit-tests-in-chrome)
+* [Debugging WebDriver Failures](#debugging-webdriver-failures)
+  * [Graphical (non-headless)](#graphical-non-headless)
+  * [Attaching a Debugger](#attaching-a-debugger)
+
+
+## Workflow
+
+### Filing Bugs
+
+**If you notice a security issue, please email 
+[arcs-team@google.com](mailto:arcs-team+security@google.com) instead of filing a bug.**
+
+TODO(arcs-team): Devise a way to track bugs internally and in the open.
```

Comment:
I think we can remove this for now and just use GH issues, and it's on us to make sure they're cross-reflected manually until we get something automated or otherwise in place.